CMakeLists.txt

```diff
@@ -191,7 +193,7 @@ if (WITH_TORCH)
   find_package(Torch REQUIRED)
   # This is annoying, torch populates all my cuda flags
   # and resets them
-  set(CMAKE_CUDA_FLAGS "")
+  # set(CMAKE_CUDA_FLAGS "")
```

Comment:
I don't remember why I did this in the first place. But did you maybe investigate the implications of commenting this out?